wpull/driver/process.py

```diff
@@ -53,8 +53,9 @@ def start(self, use_atexit=True):
         )
         self._process = yield from process_future
 
-        self._stderr_reader = asyncio.async(self._read_stderr())
-        self._stdout_reader = asyncio.async(self._read_stdout())
+        _async=getattr(asyncio, 'async')
+        self._stderr_reader = _async(self._read_stderr())
```

Comment:
I believe the right way to do this is `self._stderr_reader = asyncio.ensure_future(self._read_stderr())`